src/extractor.rs

```diff
@@ -55,7 +55,8 @@ fn choose(
     predicate: impl FnMut(&Rules, &str) -> bool,
     mut replacer: impl FnMut(&Rules, &str) -> String,
 ) -> Vec<String> {
-    let sentences_replaced_abbreviations: Vec<String> = SentenceTokenizer::<Standard>::new(text, training_data)
+    let sentences_replaced_abbreviations: Vec<String> = SentenceTokenizer::<Standard>::new(replacer(rules, text).as_ref(), training_data)
```

Comment:
I can totally see why you would want to do that. Let me think about this and report back.

Reply:
Sounds good.

Reply:
Tested. Looks good.